README.md

```diff
@@ -1,654 +1,130 @@
 
 - [Auth0's general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
 - [Auth0's code of conduct guidelines](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
-- [The Auth0 PHP SDK contribution guide](CONTRIBUTING.md)
-
-## Support + Feedback
-
-- The [Auth0 Community](https://community.auth0.com/) is a valuable resource for asking questions and finding answers, staffed by the Auth0 team and a community of enthusiastic developers
-- For code-level support (such as feature requests and bug reports), we encourage you to [open issues](https://github.com/auth0/auth0-PHP/issues) here on our repo
-- For customers on [paid plans](https://auth0.com/pricing/), our [support center](https://support.auth0.com/) is available for opening tickets with our knowledgeable support specialists
-
-Further details about our support solutions are [available on our website.](https://auth0.com/docs/support)
-
-## Vulnerability Reporting
-
-Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
-
-## What is Auth0?
 
-Auth0 helps you to:
+### Raise an issue
+To provide feedback or report a bug, [please raise an issue on our issue tracker](https://github.com/auth0/auth0-PHP/issues).
 
-- Add authentication with [multiple authentication sources](https://docs.auth0.com/identityproviders), either social like Google, Facebook, Microsoft, LinkedIn, GitHub, Twitter, Box, Salesforce (amongst others), or enterprise identity systems like Windows Azure AD, Google Apps, Active Directory, ADFS or any SAML Identity Provider.
-- Add authentication through more traditional **[username/password databases](https://docs.auth0.com/mysql-connection-tutorial)**.
-- Add support for [passwordless](https://auth0.com/passwordless) and [multi-factor authentication](https://auth0.com/docs/mfa).
-- Add support for [linking different user accounts](https://docs.auth0.com/link-accounts) with the same user.
-- Analytics of how, when, and where users are logging in.
-- Pull data from other sources and add it to the user profile through [JavaScript rules](https://docs.auth0.com/rules).
+### Vulnerability Reporting
+Please do not report security vulnerabilities on the public Github issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
 
-[Why Auth0?](https://auth0.com/why-auth0)
+---
 
-## License
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="./assets/auth0_light_mode.png"   width="150">
+    <source media="(prefers-color-scheme: dark)" srcset="./assets/auth0_dark_mode.png" width="150">
+    <img alt="Auth0 Logo" src="./auth0_light_mode.png" width="150">
```

Comment:
I'm not too sure on the banners, we uploaded the logos as they're shared across all projects so it seemed sensible to have them in a central palce. I think it maybe does make sense to upload the banners also to avoid any package managers/docs sites failing to render them